openwebif/api.py

```diff
@@ -91,7 +92,8 @@ class CreateDevice():
     def __init__(self, host=None, port=DEFAULT_PORT,
                  username=None, password=None, is_https=False,
                  prefer_picon=False, mac_address=None,
-                 turn_off_to_deep=False, source_bouquet=None):
+                 turn_off_to_deep=False, source_bouquet=None,
+                 message_timeout=None):
```

Comment:
Would rename `message_timeout` to `message_display_timeout` so it is a bit more obvious of the intent.